Gemfile.template

```diff
@@ -2,6 +2,18 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
+
+if Dir.exist?("../logstash-extra")
+  Dir.open("../logstash-extra").each do |entry|
+    next if entry.start_with?('.')
+    path = "../logstash-extra/#{entry}"
+    next unless File.directory?(path)
+    name = entry =~ /^(.+)-logstash$/ ? $1 : entry 
+    puts "Loading development gem: '#{name}' from '#{path}'"
+    gem name, :path => path
+  end
+end
+
```

Comment:
I don't think it would be a bug in x-pack but in core, I think we only have the risk to run into that when we actually test the runner because it will load code and execute events on the x-pack. 

Might be a matter of only mock these interaction when running the test sutie?

Let me do a few local test and I will report back, if its that OK with you?